compiler/record-minmax-thread-safety-test/CMakeLists.txt

```diff
@@ -2,6 +2,11 @@ if(NOT ENABLE_TEST)
     return()
 endif(NOT ENABLE_TEST)
 
+# Disable the test if record-minmax-for-thread-test does not exist
+if (NOT TARGET record-minmax-for-thread-test)
+    return()
```

Comment:
optional)
can you add a message to show that `record-minmax-for-thread-test` is temporary disabled?